benchmarks/upload_local.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import logging
+
+logging.basicConfig(level=logging.ERROR)
+
+
+import hub
+from benchmark_utils import timer, parametrize
+from tfds_utils import datasets, download, DATASETS_DIR, get_hub_ds_path
+import posixpath
+
+
+username = open("username.txt", "r").read()
+
+
+@parametrize(dataset=datasets)
+def upload_local(dataset):
+    download(dataset)
+    ds_local = hub.Dataset(get_hub_ds_path(dataset))
+    cloud_path = f"hub://{username}/{dataset}"
```

Comment:
please refer to our [contributing md](https://github.com/activeloopai/Hub/blob/release/2.0/CONTRIBUTING.md) for the proper way to do this

Reply:
(fixtures)